athlete/src/com/lambdaschool/solution/BaseballAthleteProcessor.java

```diff
@@ -0,0 +1,9 @@
+package com.lambdaschool.solution;
+
+public class BaseballAthleteProcessor implements Processor
+{
+    @Override
+    public void displayAthlete() {
+        System.out.println("************" + "\nBaseball Athlete\n" + "************");
```

Comment:
just remember that these "*"'s are repeating in a lot of your files, it could be abstracted into another class and then pass the impl class object into that to display them all.